packages/api-server/api_server/routes/delivery_alerts.py

```diff
@@ -17,7 +17,9 @@ async def sub_delivery_alerts(_req: SubscriptionRequest):
     return get_rmf_events().delivery_alerts.pipe(rxops.filter(lambda x: x is not None))
 
 
-@router.post("/{delivery_alert_id}/response", response_model=DeliveryAlert)
+@router.post(
+    "/{delivery_alert_id}/response", response_model=DeliveryAlert, deprecated=True
```

Comment:
Add docs for alternatives for those who are using this api.

Reply:
Good point, added that and also deprecated the router,
![image](https://github.com/user-attachments/assets/7b32dd4f-91bb-4dbc-a0aa-73549e5f010b)